doc/source/ray-air/check-ingest.rst

```diff
@@ -157,6 +157,52 @@ See the following examples for clarification:
         :start-after: __config_5__
         :end-before: __config_5_end__
 
+Shuffling Data
+~~~~~~~~~~~~~~
+
+AIR offers several options for per-epoch shuffling, including *global
+(whole-dataset) shuffling*  and *local (per-shard) shuffling*.
+
+.. tabbed:: Global Shuffling
```

Comment:
Make this the second tab (de-emphasize)?

Reply:
Switched order.